.gitlab-ci.yml

```diff
@@ -23,11 +23,11 @@ stages:
 
 .pr_workflow:
   rules:
-    - if: '$CI_COMMIT_BRANCH != "master" && $BUILD_DEPS != "ON"' #run only if ...
+    - if: '$CI_COMMIT_BRANCH != "main" && $CI_COMMIT_BRANCH != "develop" && $BUILD_DEPS != "ON"' #run only if ...
 
-.master_workflow:
+.main_workflow:
```

Comment:
This might need to coincide with a change in uberenv. @adrienbernede  , who may not be available to answer atm?

Reply:
Processing...

Reply:
What I will do:
- Sign-off this branch (LGTM) so that it gets imported to Gitlab.
- Trigger a pipeline that mimics "main" or "develop" case on Gitlab to vet that it works fine. (we can manually trigger a pipeline, and setting BUILD_DEPS == "ON" will mimic the main workflow.)
- Push the corresponding change in Uberenv CI, so that it points to develop instead of master.